README.md

```diff
@@ -0,0 +1,86 @@
+# Overture Iobio Components
+
+- A set of reusable React components and command line tools for integrating the visualization and analysis software originally created by [IOBIO](https://iobio.io/).
+
+- This project is part of the [Overture](https://www.overture.bio/) research software ecosystem. See our [related products](#related-products) for more information on how Overture is helping organize data and enable discovery.
+
+</br>
+
+> 
+> <div>
+> <img align="left" src="ov-logo.png" height="90"/>
```

Comment:
```suggestion
> <img align="left" src="ov-logo.png" height="50"/>
```

Reply:
This will keep the Overture Logo from interfering with the Repository structure title when viewing from the repo page